scripts/linters/pylint_extensions.py

```diff
@@ -1484,31 +1484,63 @@ class RestrictedImportChecker(checkers.BaseChecker):
             'domain layer and controller layer respectively.'),
     }
 
-    def visit_import(self, node):
-        """Visits every import statement in the file.
+    options = (
+        (
+            'forbidden-imports',
+            {
+                'default': ('core.storage:core.domain',),
```

Comment:
Why do we need this default value? (Note: this is still project-specific information.)

Reply:
Nope, removed.